### PR TITLE
Update Português Brasileiro locale

### PR DESCRIPTION
**Pull request by rossignolli**

### CODE REVIEW

**Comment by Lhoerion** [comment]
You forgot to add comma on the line before your new additions.

**Comment by Lhoerion** [comment]
There's still like ~13 properties missing, would be nice to add them to along with making it uniform with template file (`en.json`).
For your work verification you can use npm command `npm run test -- langs/pt_br.json` to test if it passes checks.

**Comment by Lhoerion** [comment]
Thanks for the contribution!
Besides adding missing properties and reordering them, you also changed some existing translations, are you sure that they were incorrect?

**Comment by rossignolli** [comment]
I made some improvements to all of them based on en.json

**Comment by Lhoerion** [comment]
Yes, but I see that you've changes some already existing translations, were they incorrect?

**Comment by rossignolli** [comment]
They were not incorrect, but out of order based on en.json.

File: langs/pt_br.json [hunk 1]
```diff
@@ -1,66 +1,79 @@
...
+    "SERVER_NAME": "Nome do Servidor",
     "PLAYERS": "Jogadores",
     "GAMEMODE": "Modo de Jogo",
     "PING": "Latência",
-    "WEBSITE": "Site",
-    "FAVORITE_SERVERS": "Servidores favoritos",
+    "WEBSITE": "Website",
+    "FAVORITE_SERVERS": "Servidores Favoritos",
     "LANGUAGE": "Idioma",
-    "ENTER_PASSWORD": "Digite a senha",
```
Comment (Lhoerion) [inline langs/pt_br.json:24]: This was changed for example, was it incorrect?

Comment (Lhoerion) [reply langs/pt_br.json:24]: Ok, thanks for clarification, I had to make sure what was the changes for

**Pull request merged**